pkg/daemon/ceph/test/info.go

```diff
@@ -1,13 +1,24 @@
 package test
 
 import (
+	"github.com/coreos/pkg/capnslog"
 	"io/ioutil"
 	"os"
 	"path"
 )
 
+var (
+	logger = capnslog.NewPackageLogger("github.com/rook/rook", "cephtest")
+)
+
 func CreateConfigDir(configDir string) {
```

Comment:
These methods are for testing right? I wonder if it is best to panic if we find an error here rather than just printing an error. Printing the error isn't likely to get the user's attention, and it won't change the results of the test. @travisn @galexrt ?

Reply:
Yes. That is true. I'll return this error and give appropriate checks from the _test.go files.